scripts/upgrade

```diff
@@ -18,6 +18,12 @@ serviceuser=$(ynh_app_setting_get $app serviceuser)
 path=$(ynh_app_setting_get $app path)
 port=$(ynh_app_setting_get $app port)
 
+# stop rocketchat and remove program dir
+sudo systemctl stop rocketchat
+
+sudo rm -rf $final_path
```

Comment:
if you rm final_path before backup a failing upgrade may destroy your installation.

Reply:
But the Backup-Function wasn't there originally, or did I oversee it? 
Nevertheless,  as far as I can see your latest update solved the problem anyway, so I think we can close the PR